docs/Installation.rst

```diff
@@ -1,90 +1,224 @@
+###############
 Installing Lago
+###############
+
+Lago is officially supported and tested on Fedora 24+ and CentOS 7.3
+distributions. However, it should be fairly easy to get it running on
+debian variants.
+
+As Lago requires libvirtd installed and several group permissions,
+it cannot be installed solely via ``pip``. For that reason the
+recommended method of installation is using the RPM. The easiest way, is
+to use the `Install script`_ which we test and verify regularly [2]_.
+
+
+pip
+===
+
+1. Install system package dependencies, this may vary according to your
+   distribution:
+
+   A. CentOS 7.3+
+
+   .. code:: bash
+
+        $ yum install -y epel-release centos-release-qemu-ev
+        $ yum install -y libvirt libvirt-devel libguestfs-tools \
+            libguestfs-devel gcc libffi-devel openssl-devel \
+            qemu-kvm-ev
+
+
+   B. Fedora 24+
+
+   .. code:: bash
+
+       $ yum install -y libvirt libvirt-devel libguestfs-tools \
+           libguestfs-devel gcc libffi-devel openssl-devel qemu-kvm
+
+   C. Debian / Ubuntu - *TO-DO*
+   D. ArchLinux - *TO-DO*
+
+2. Install libguestfs python bindings, as they are not available on PyPI [3]_:
```

Comment:
The disadvantages of a system package:
1. Virtualenv - you need to instruct users to use --with-system-packages, which breaks the concept of a virtualenv. 
2. We tie ourselves to a distro specific package, while it might not be available on every distro(though I agree in this case - if someone built 'libguestfs' package which we need anyways, he would probably build the python binding also, but that is not promised).
This is our only Python system package that we require. Without it you can really install Lago in a virtualenv, without any hacks.

In my opinion,  for pip instructions - we should strive to install whatever is Python in the Python way of things - pip, virtualenv, tox, etc. 

With all that being said, it really doesn't matter - if someone wants to use the system-package he can always do it. I can add it to the notes.